pkg/operator/controller/controller_dns_configmap.go

```diff
@@ -40,7 +40,7 @@ var corefileTemplate = template.Must(template.New("Corefile").Funcs(template.Fun
         {{- with $tls := .TransportConfig.TLS }}
         {{- with $serverName := $tls.ServerName }}
         tls_servername {{$serverName}}
-        tls {{- with $cm := $tls.CABundle.Name}} /etc/pki/{{$serverName}}-{{ index $.CABundleRevisionMap $cm }}/{{ $.CABundleFileName }}{{end}}
+        tls {{- with $.CABundleRevisionMap }}{{- with $cm := (index $.CABundleRevisionMap $tls.CABundle.Name) }} /etc/pki/{{$serverName}}-{{ $cm }}/{{ $.CABundleFileName }}{{end}}{{end}}
```

Comment:
Should the entire line be removed or does `tls` with nothing following it, work ok?

Reply:
Similarly here, `tls` alone on one line is a valid configuration. In our case a cluster admin can configure one or none configmaps; none indicating that they want to use the system certs for the upstream server validation.